pkgs/misc/vim-plugins/update.py

```diff
@@ -218,16 +220,17 @@ def get_current_plugins() -> List[Plugin]:
 
 
 def prefetch_plugin(
-    user: str, repo_name: str, alias: str, cache: "Cache"
+    user: str, repo_name: str, alias: str, cache: "Cache" = None
```

Comment:
Should this one not be catched by mypy?

```suggestion
    user: str, repo_name: str, alias: str, cache: "Optional[Cache]" = None
```

Reply:
For some reason mypy doesn't seem to care whether this is done for optional arguments but I'll go and make this change.